src/style.css

```diff
@@ -1 +1,139 @@
-/* empty source */
+html {
+  --main-text-colour: rgba(6, 11, 53, 1);
+  --second-text-colour: rgba(97, 96, 112, 1);
+  --button-colour: rgba(0, 172, 220, 1);
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
```

Comment:
if you set font weight here you dont need to set it up in every text block, it will be inherit